### PR TITLE
VACMS-11973: Updates caption title for alert and promo blocks Views.

### LINKED ISSUE #11973: Incorrect Table Caption Displayed on Alert and Promo Block Views
## Description
During [Home Page Hardening](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11160), we created, and later removed, a new View display to allow Home Page Managers to manage newly created home page blocks. In doing so, we inadvertently added incorrect table captions for both the Alerts and Promos displays as well.

image (1)

Update the Table caption in both Alerts and Promos View displays per the [Views Style Guide](https://va-gov.atlassian.net/wiki/spaces/VAGOV/pages/1740472321/CMS+Views+Style+Guide)

## Acceptance Criteria
- [ ]  [Promo](/admin/content/promos) table caption reads `List of Promo blocks`
- [ ]  [Alert](/admin/content/alerts) table caption reads `List of Alert blocks`

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [x] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by dsasser**
## Description

Closes #11973

### Before
image (1)

### After
Screen Shot 2022-12-15 at 12 50 52 PM
Screen Shot 2022-12-15 at 12 52 01 PM

## QA steps
As a content admin:
- [ ]  [Promo](https://cms-b9kbil3n1pepsl1xw1jupasprxbaukl3.ci.cms.va.gov//admin/content/promos) table caption reads `List of Promo blocks`
- [ ]  [Alert](https://cms-b9kbil3n1pepsl1xw1jupasprxbaukl3.ci.cms.va.gov//admin/content/alerts) table caption reads `List of Alert blocks`

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [x] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### CODE REVIEW

File: config/sync/views.view.va_blocks_admin.yml [hunk 1]
```diff
@@ -4,22 +4,16 @@ status: true
 dependencies:
   config:
     - block_content.type.alert
-    - block_content.type.benefit_promo
```
Comment (dsasser) [inline config/sync/views.view.va_blocks_admin.yml:7]: I was surprised to see these changes, however these config dependency removals are correct. I am unsure why Drupal didn't calculate them correctly before (when I removed the 3rd display) during #11160. But nonetheless they are relevant only to V2 Home page blocks, which are no longer part of this View.

File: config/sync/views.view.va_blocks_admin.yml [hunk 2]
```diff
@@ -750,33 +743,23 @@ display:
           row_class: ''
           default_row_class: true
           columns:
-            info: info
```
Comment (dsasser) [inline config/sync/views.view.va_blocks_admin.yml:753]: I had to override Alerts display for the table to have unique caption for each display. This is causing the wall of changes you see below.

File: config/sync/views.view.va_blocks_admin.yml (hunk 1 above)
Comment (swirtSJW) [reply config/sync/views.view.va_blocks_admin.yml:7]: Have seen on more than one occasion where drupal need a second flush to resolve its sanitation correctly.  Always good to look at it as you have.

**Review by swirtSJW — APPROVED** [review]
Tests out good.  All looks right and 508 improved.  Thank you.

**Pull request merged**